Makefile

```diff
@@ -98,13 +100,25 @@ vendor:
 	cp -rf --no-preserve=mode "$${_src}" "$${_dst}"
 .PHONY: vendor
 
+# GENERATE_YAML_ENV_VARS is the list of all environment variables expected by generate-yaml.sh
+# to build example manifests for clusterctl. The list of environment variables here should contain
+# the environment variables expected in generate-yaml.sh
+GENERATE_YAML_ENV_VARS := -e CLUSTER_NAME -e SERVICE_CIDR -e CLUSTER_CIDR -e CAPV_YAML_VALIDATION \
```

Comment:
Hi @andrewsykim,

By the way, I forgot to say, thank you for teaching me about this feature of Docker. I didn't realize you could use `-e` to set an env var without also the value. This is nice.